server/app/controllers/applicant/ApplicantProgramBlocksController.java

```diff
@@ -413,6 +485,45 @@ private CompletionStage<Result> renderErrorOrRedirectToNextBlock(
                           ApplicantQuestionRendererParams.ErrorDisplayMode.DISPLAY_ERRORS))));
     }
 
+    if (featureFlags.isEsriAddressCorrectionEnabled(request)
+        && thisBlockUpdated.hasAddressWithCorrectionEnabled()) {
+
+      ApplicantQuestion applicantQuestion =
+          applicantService.getFirstAddressCorrectionEnabledApplicantQuestion(thisBlockUpdated);
+
+      AddressQuestion addressQuestion = applicantQuestion.createAddressQuestion();
```

Comment:
**nitpick:** `applicantQuestion` is immediately used and then not consumed again. condensing this into a single statement doesn't make the reader wonder if it's reused further down
```suggestion
      AddressQuestion addressQuestion =
          applicantService
              .getFirstAddressCorrectionEnabledApplicantQuestion(thisBlockUpdated)
              .createAddressQuestion();
```

Reply:
done